docs/docs/adding-pagination.md

```diff
@@ -60,57 +60,51 @@ export const blogListQuery = graphql`
 const path = require("path")
 const { createFilePath } = require("gatsby-source-filesystem")
 
-exports.createPages = ({ graphql, actions }) => {
+exports.createPages = async ({ graphql, actions }) => {
```

Comment:
Going through this I realized this is extending the blog starter. Maybe we ought to swap the gatsby-starter-blog `gatsby-node` file to use async/await as well.. though that could probably come in another PR.

Reply:
I see. That's in another repo https://github.com/gatsbyjs/gatsby-starter-blog so I'll submit the PR over there.

Reply:
Actually, it's not that one.